Tests/Editor/CCUTest.cs

```diff
@@ -0,0 +1,86 @@
+﻿using System;
+using System.Collections.Generic;
+using System.IO;
+using ConditionalCompilation;
+using NUnit.Framework;
+using UnityEngine;
+
+namespace UnityEditor.Experimental.EditorVR.Tests
+{
+	[InitializeOnLoad]
+	public class CCUTest
+	{
+		[Test]
+		public void TestCCU()
+		{
+			var buildTargetGroup = EditorUserBuildSettings.selectedBuildTargetGroup;
+			Assert.IsTrue(PlayerSettings.GetScriptingDefineSymbolsForGroup(buildTargetGroup).Contains(ConditionalCompilationUtility.kEnableCCU));
+		}
+
+		[Test]
+		public void NoDependencyTest()
+		{
+			// TODO: Find a better way to collect dependencies
+			var dependencyPath = Path.Combine(EditorApplication.applicationContentsPath, "Managed");
+			var extensionsPath = Path.Combine(EditorApplication.applicationContentsPath, "UnityExtensions");
+
+			var references = new List<string>
+			{
+				Path.Combine(dependencyPath, "UnityEngine.dll"),
+				Path.Combine(dependencyPath, "UnityEditor.dll"),
+
+				Path.Combine(Path.Combine(Path.Combine(extensionsPath, "Unity"), "GUISystem"), "UnityEngine.UI.dll"),
+				Path.Combine(Path.Combine(Path.Combine(Path.Combine(extensionsPath, "Unity"), "GUISystem"), "Editor"), "UnityEditor.UI.dll"),
+				Path.Combine(Path.Combine(Path.Combine(Path.Combine(extensionsPath, "Unity"), "EditorTestsRunner"), "Editor"), "nunit.framework.dll"),
+				Path.Combine(Path.Combine(Path.Combine(Path.Combine(extensionsPath, "Unity"), "EditorTestsRunner"), "Editor"), "UnityEditor.EditorTestsRunner.dll")
+			};
+
+			//GetAllFiles(EditorApplication.applicationContentsPath, references, "*.dll"); // Need to weed out unmanaged dlls
+			//GetAllFiles(dependencyPath, references, "*.dll"); // Error on loading ICsharpCode.NRefactory
+
+			var sources = new List<string>();
+			GetAllFiles(Application.dataPath, sources, "*.cs");
+
+			// TODO: Find a better way to collect #defines
```

Comment:
defines = EditorUserBuildSettings.activeScriptCompilationDefines - defines that come from CCU (create a separate property or method for those)